iidm/iidm-tck/src/test/java/com/powsybl/iidm/network/tck/util/AbstractSwitchPredicatesTest.java

```diff
@@ -0,0 +1,307 @@
+            .add();
+        vl.getNodeBreakerView().newBreaker()
+            .setId("B_L1_2")
+            .setNode1(4)
+            .setNode2(7)
+            .setOpen(false)
+            .add();
+        vl.getNodeBreakerView().newBreaker()
+            .setId("B_G1")
+            .setNode1(5)
+            .setNode2(8)
+            .setOpen(true)
+            .add();
+        vl.getNodeBreakerView().newBreaker()
+            .setId("B_G2")
+            .setNode1(6)
+            .setNode2(9)
+            .setOpen(true)
+            .setFictitious(true)
+            .add();
+        vl.getNodeBreakerView().newBreaker()
+            .setId("B0")
+            .setNode1(7)
+            .setNode2(17)
+            .setOpen(false)
+            .add();
+        vl.getNodeBreakerView().newBreaker()
+            .setId("B1")
+            .setNode1(8)
+            .setNode2(11)
+            .setOpen(true)
+            .add();
+        vl.getNodeBreakerView().newBreaker()
+            .setId("B2")
+            .setNode1(9)
+            .setNode2(12)
+            .setOpen(false)
+            .setFictitious(true)
+            .add();
+        vl.getNodeBreakerView().newBreaker()
+            .setId("B3")
+            .setNode1(7)
+            .setNode2(8)
+            .setOpen(false)
+            .add();
+        vl.getNodeBreakerView().newSwitch()
+            .setId("B4")
+            .setNode1(8)
+            .setNode2(9)
+            .setOpen(false)
+            .setKind(SwitchKind.LOAD_BREAK_SWITCH)
+            .add();
+        vl.getNodeBreakerView().newBreaker()
+            .setId("B5")
+            .setNode1(17)
+            .setNode2(10)
+            .setOpen(false)
+            .add();
+
+        // Disconnectors
+        vl.getNodeBreakerView().newDisconnector()
+            .setId("D0")
+            .setNode1(0)
+            .setNode2(10)
+            .setOpen(true)
+            .add();
+        vl.getNodeBreakerView().newDisconnector()
+            .setId("D1")
+            .setNode1(1)
+            .setNode2(10)
+            .setOpen(false)
+            .add();
+        vl.getNodeBreakerView().newDisconnector()
+            .setId("D2")
+            .setNode1(0)
+            .setNode2(11)
+            .setOpen(false)
+            .add();
+        vl.getNodeBreakerView().newDisconnector()
+            .setId("D3")
+            .setNode1(1)
+            .setNode2(11)
+            .setOpen(true)
+            .add();
+        vl.getNodeBreakerView().newDisconnector()
+            .setId("D4")
+            .setNode1(2)
+            .setNode2(12)
+            .setOpen(false)
+            .add();
+        vl.getNodeBreakerView().newDisconnector()
+            .setId("D5")
+            .setNode1(3)
+            .setNode2(12)
+            .setOpen(true)
+            .add();
+        return network;
+    }
+
+    @Test
+    public void testNonFictionalClosedBreakers() {
+        // Network
+        Network network = createNetwork();
```

Comment:
To avoid recreating the network for each test (it is not altered by the tests), the `createNetwork` can be called in a static method annotated with `@BeforeAll`.

Reply:
done